src/components/TeacherViews/LessonDownloads.view.tsx

```diff
@@ -288,74 +293,87 @@ export function LessonDownloads(props: LessonDownloadsProps) {
           />
           <Hr $color={"grey60"} $mt={24} />
         </Box>
-        {!isDownloadSuccessful && (
-          <ResourcePageLayout
-            page={"download"}
-            errors={form.errors}
-            handleToggleSelectAll={handleToggleSelectAll}
-            selectAllChecked={selectAllChecked}
-            header="Download"
-            showNoResources={showNoResources}
-            showLoading={isLocalStorageLoading}
-            email={emailFromLocalStorage}
-            school={schoolNameFromLocalStorage}
-            schoolId={schoolIdFromLocalStorage}
-            setSchool={setSchool}
-            showSavedDetails={shouldDisplayDetailsCompleted}
-            onEditClick={handleEditDetailsCompletedClick}
-            register={form.register}
-            control={form.control}
-            showPostAlbCopyright={!isLegacyDownload}
-            resourcesHeader="Lesson resources"
-            triggerForm={form.trigger}
-            apiError={apiError}
-            hideSelectAll={Boolean(expired)}
-            updatedAt={updatedAt}
-            withHomeschool={true}
-            cardGroup={
-              !showNoResources && (
-                <DownloadCardGroup
-                  control={form.control}
-                  downloads={downloadsFilteredByCopyright}
-                  hasError={form.errors?.resources ? true : false}
-                  triggerForm={form.trigger}
-                />
-              )
-            }
-            cta={
-              <LoadingButton
-                type="button"
-                onClick={
-                  (event) => void form.handleSubmit(onFormSubmit)(event) // https://github.com/orgs/react-hook-form/discussions/8622}
-                }
-                text={"Download .zip"}
-                icon={"download"}
-                isLoading={isAttemptingDownload}
-                disabled={
-                  hasFormErrors ||
-                  noResourcesSelected ||
-                  showNoResources ||
-                  (!form.formState.isValid && !localStorageDetails)
-                }
-                loadingText={"Downloading..."}
+        {(() => {
+          if (
+            user &&
+            lesson.isDownloadRegionRestricted &&
+            !user.publicMetadata.owa?.isRegionAuthorised
+          ) {
+            return <LessonDownloadRegionBlocked />;
+          }
+
+          if (isDownloadSuccessful) {
+            return (
+              <DownloadConfirmation
+                lessonSlug={lessonSlug}
+                lessonTitle={lessonTitle}
+                unitSlug={unitSlug}
+                unitTitle={unitTitle}
+                programmeSlug={programmeSlug}
+                data-testid="downloads-confirmation"
+                isCanonical={props.isCanonical}
+                nextLessons={lesson.nextLessons}
+                onwardContentSelected={onwardContentSelected}
+                isSpecialist={isSpecialist}
               />
```

Comment:
I was initially going to create a whole other page, but bringing in the breadcrumbs and everything else would have made this a lot more work. Adding a third state to the downloads page made a lot more sense.